src/program/posix.lalrpop

```diff
@@ -226,7 +231,12 @@ File: ast::Redirect = {
 //     },
 // }
 
+// pub Word: ast::Word = {
+//     <w: "WORD"> => ast::Word(w.into()),
+//     "$" <v: "WORD"> => ast::Word(var(v).unwrap_or(format!("${}", v))),
+// }
```

Comment:
Question for quotes too, do we try to parse these separately, or handle as `"WORD"` and `Word(String)`?